packages/apps/web-viewer-test/src/components/home/IModelBankHome.tsx

```diff
@@ -0,0 +1,158 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { BrowserAuthorizationClientConfiguration } from "@bentley/frontend-authorization-client";
+import { IModelBankClient } from "@bentley/imodelhub-client";
+import {
+  FitViewTool,
+  IModelApp,
+  ScreenViewport,
+  StandardViewId,
+} from "@bentley/imodeljs-frontend";
+import { ColorTheme } from "@bentley/ui-framework";
+import { IModelBackendOptions, Viewer } from "@itwin/web-viewer-react";
+import React, { useEffect, useState } from "react";
+
+import { history } from "../routing";
+import { Header } from ".";
+import styles from "./Home.module.scss";
+
+/**
+ * Test a viewer that is connected to an iModelBank
+ * @returns
+ */
+export const IModelBankHome: React.FC = () => {
+  const [loggedIn, setLoggedIn] = useState(
+    (IModelApp.authorizationClient?.hasSignedIn &&
+      IModelApp.authorizationClient?.isAuthorized) ||
+      false
+  );
+  const [iModelId, setIModelId] = useState(
+    process.env.IMJS_AUTH_CLIENT_IMODEL_ID
+  );
+  const [contextId, setContextId] = useState(
+    process.env.IMJS_AUTH_CLIENT_CONTEXT_ID
+  );
+
+  const authConfig: BrowserAuthorizationClientConfiguration = {
+    scope: process.env.IMJS_AUTH_CLIENT_SCOPES ?? "",
+    clientId: process.env.IMJS_AUTH_CLIENT_CLIENT_ID ?? "",
+    redirectUri: process.env.IMJS_AUTH_CLIENT_REDIRECT_URI ?? "",
+    postSignoutRedirectUri: process.env.IMJS_AUTH_CLIENT_LOGOUT_URI,
+    responseType: "code",
+    authority: process.env.IMJS_AUTH_AUTHORITY,
+  };
+
+  const backend: IModelBackendOptions = {
+    customBackend: {
+      rpcParams: {
+        info: { title: "sample-backend", version: "1.0" },
+        uriPrefix: "https://dev-imodelbank.bentley.com/imodeljs",
```

Comment:
is the iModel Bank deployed to prod anywhere?
a little hesitant to have a reference to non prod envs

Reply:
there is no prod imodelbank AFAIK. Its just a test environment for on premise cloud. This would not be accessible anyways. Does it matter?

Reply:
gotcha, its fine then.